app/lib/bk/compat/parsers/bitrise/steps.rb

```diff
@@ -28,6 +59,18 @@ def translate_git_clone(_inputs)
           '# No need for cloning, the agent takes care of that'
         end
 
+        def translate_git_tag(inputs)
+          return '# Invalid git-tag step configuration!' unless inputs.include?('tag')
+
+          cmd_all_params = "git tag -fa #{inputs['tag']} -m #{inputs['tag_message']}"
+          cmd_no_message = "git tag -fa #{inputs['tag']}"
+
+          [
+            inputs['tag_message'].nil? ? cmd_no_message : cmd_all_params,
```

Comment:
in the [original code there are quotes around the tag message](https://github.com/Itelios/bitrise-steps-git-tag/blob/master/step.sh#L5) to ensure that, if not present it uses an empty string as the message and avoid the need for two different branches:

```suggestion
          [
            "git tag -fa #{inputs['tag']} -m \"#{inputs['tag_message']}\""
```

Reply:
Oh yep one I missed 😮‍💨 - kept the initial ternary just on the case of no tag message; as the quotes would wrap a `nil`!

Reply:
that is weird,  I checked the behaviour in irb and it shouldn't be that way:

```
> inputs
=> {"tag"=>"v0.1.0"}
> "git tag -fa #{inputs['tag']} -m \"#{inputs['tag_message']}\""
=> "git tag -fa v0.1.0 -m \"\""
```